lib/preboots/index.js

```diff
@@ -9,7 +9,8 @@ module.exports = function preboot(app, options, next) {
   app.preboot(require('slay-log')({
     transports: [
       new (winston.transports.Console)({
-        raw: app.env !== 'local'
+        raw: app.env !== 'local',
+        timestamp: true
```

Comment:
If you are fiddling with `winston` might be good time to also upgrade to `slay@2` which upgrades to `winston@3`. Non-blocking though.